docs/other/fontsobject.rst

```diff
@@ -133,9 +151,9 @@ This function will return an array of :ref:`fontobject` based on the Postscript
 **Parameters**
 
 ====================  ========================================================
-postscriptName          A string containing the Postscript Name of the font.
+postscriptName          A string containing the PostScript Name of the font.
 ====================  ========================================================
 
 **Returns**
 
-An array of fontObjects.
+An array of :ref:`fontobject`.
```

Comment:
Prefer the use of labels for references so the sentences can read correctly while still linking to the Font object entry.
```suggestion
Array of :ref:`Font objects<fontobject>`; read-only.
```